assets/components/directDebit/directDebitActions.js

```diff
@@ -50,12 +54,13 @@ function payDirectDebitClicked(callback: Function): Function {
   return (dispatch: Function, getState: Function) => {
 
     const {
-      bankSortCode,
-      bankAccountNumber,
+      sortCodeArray,
+      bankAccountNumber: accountNumber,
```

Comment:
I thought we would drop the `bank` from `bankAccountNumber`

Reply:
I would write something like this:
```
 const {
   sortCodeArray,
 accountNumber,
accountHolderName,
       accountHolderConfirmation,	       accountHolderConfirmation,
     } = getState().page.directDebit;	     } = getState().page.directDebit;
```
Since you are using destructuring assigments.

Reply:
Yup I meant to, must have missed this one.